SecondSight/config.py

```diff
@@ -2,6 +2,22 @@
 
 import sys
 import json
+import os
+
+
+def loadConfig():
+    if not os.path.exists('config.json'):
+        print('PLEASE MAKE A CONFIG FILE')
```

Comment:
We shouldn't print things

But also, if there is no config data, just redirect the web page to /config